<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj zakres mas losowanych mMin,mMax(1000t)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wprowadz wlasne dane</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mMinE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.01</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="iCialE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xMinE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>200</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="VxMaxE">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj ilosc cial (liczba calkowita wieksza niz 0)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj zakres losowanych yMin,yMax(m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="VxMinE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj zakres losowanych VyMin,VyMax(m/s)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yMinE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>200</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj zmiane czasu(s) (liczba dodatnia i mniejsza od czasu)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj zakres losowanych xMin,xMax(m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="VyMinE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yMaxE">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>400</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="VyMaxE">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj zakres losowanych VxMin,VxMax(m/s)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mMaxE">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dTimeE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xMaxE">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>400</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj czas (s)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wczytaj usawienia z pliku</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
